documentation/docs/pmm-upgrade/migrating_from_pmm_2.md

```diff
@@ -19,17 +19,35 @@ Before upgrading to PMM 3, ensure your PMM 2 Server is running the latest versio
 
 === "Automated upgrade (Recommended)"
     Use this upgrade script for a simplified migration process:
+    { .power-number}
 
-    1. Download and run the following script to start the upgrade. The `-b` flag ensures your data is backed up before the upgrade.
+    1. Download and run the [automated upgrade script](https://raw.githubusercontent.com/percona/pmm/0ad7c05ae253948c779e48ff7976cb5c982af688/get-pmm.sh) to start the upgrade. The `-b` flag creates a backup of your PMM2 instance to ensure that your data is backed up before the upgrade.
 
         ```sh
         ./get-pmm.sh -n <container-name> -b
         ```
+    2. Note the backup volume name displayed during the upgrade (e.g., `pmm-data-2025-01-16-165135`) so that you can restore this backup if needed.
 
-    2. Check additional script options:
+    3. Check additional script options:
         ```sh
         ./get-pmm.sh -h
         ```
+    !!! note alert alert-primary "Restore PMM 2 backup"
+        If you need to revert to the PMM 2 instance, restore the backup created above:
+        { .power-number}
+
+
+        1. Stop the PMM 3 container:
+            ```sh
+            docker stop pmm-server
+            ```
+        2. Start a PMM 2 container using the backup volume, replacing `<backup-volume-name>` (e.g., `pmm-data-2025-01-16-165135`) with your actual backup volume name:
+
+            ```sh
+            docker run -d -p 443:443 --volume <backup-volume-name>:/srv --name pmm-server --restart always percona/pmm-server:2.44.0
+            ```
+        3. Verify that your PMM 2 instance is running correctly and all your data is accessible.
+
 
 === "Manual upgrade from PMM 2 with Docker volume"
```

Comment:
Upgrade of Helm chart:
1. helm repo update percona
2. helm show values percona/pmm > values.yaml
3. Update values.yaml to have similar configuration with PMM 2
4. kubectl exec pmm-0 -- supervisorctl stop all
5. kubectl exec pmm-0 -- chown -R pmm:pmm /srv 
6. helm upgrade pmm -f values.yaml --set secret.create=false \
--set secret.name=pmm-secret \
percona/pmm
7. in case k8s is not upgraded try `kubectl delete pod pmm-0`

Reply:
Manual upgrade of podman:

1. podman pull percona/pmm-server:3
2. podman exec pmm-server supervisorctl stop all
3. podman exec pmm-server chown -R pmm:pmm /srv
4. Remove ~/.config/systemd/user/pmm-server.service
5. Follow steps from https://github.com/percona/pmm/blob/Release-3-0-0-GA/documentation/docs/install-pmm/install-pmm-server/baremetal/podman/index.md